src/HotChocolate/Core/src/Types.Scalars/EmailAddressType.cs

```diff
@@ -1,6 +1,5 @@
 using System.Text.RegularExpressions;
 using HotChocolate.Language;
-using HotChocolate.Types.Scalars;
 
```

Comment:
Do these still compile without `HotChocoalte.Types.Scalars`? I mean, the resources are still in this package right?